src/GraphQL.Tests/AssertionExtensions.cs

```diff
@@ -1,21 +1,23 @@
+using System.Text.Json;
 using Shouldly;
 
 namespace GraphQL.Tests
 {
     public static class AssertionExtensions
     {
-        public static void ShouldBeCrossPlat(this string a, string b, string customMessage)
-        {
-            var aa = a?.Replace("\r\n", "\n").Replace("\\r\\n", "\\n");
-            var bb = b?.Replace("\r\n", "\n").Replace("\\r\\n", "\\n");
-            aa.ShouldBe(bb, customMessage);
-        }
+        public static void ShouldBeCrossPlat(this string actual, string expected, string customMessage)
+            => Normalize(actual).ShouldBe(Normalize(expected), customMessage);
 
-        public static void ShouldBeCrossPlat(this string a, string b)
+        public static void ShouldBeCrossPlat(this string actual, string expected)
+            => Normalize(actual).ShouldBe(Normalize(expected));
+
+        public static void ShouldBeCrossPlatJson(this string actualJson, string expectedJson)
         {
-            var aa = a?.Replace("\r\n", "\n").Replace("\\r\\n", "\\n");
-            var bb = b?.Replace("\r\n", "\n").Replace("\\r\\n", "\\n");
-            aa.ShouldBe(bb);
+            using var actualJsonDoc = JsonDocument.Parse(Normalize(actualJson));
```

Comment:
By parse/serilize you wanted to make sure if it is a valid json inside?

Reply:
This is just to ignore any differences in whitespace/formatting. 

Pretty much all the test bases do a similar thing. They have an `Assert` method that has an argument for the json expected. It turns that into an ExecutionResult then back into a string (removing any formatting), then runs the graph query and writes the actual result to a string too. Then compares them.